### DISCUSSION TITLE
Toolbar_Group for Quickbar_Selection_Toolbar?

### DISCUSSION THREAD

**Discussion by GrtFS** [tinymce/tinymce#8997] [General]
Is there any way in TINYmce 6.6 to emulate a 'toolbar_groups' button group, but for a quickbars_selection_toolbar?

When I make a Group, it works fine in the Toolbar, but not the Quickbar. (Just doesn't appear).

Is this possible or is there a workaround?

Any help much appreciated.

**Comment by mrinasugosh** [comment]
The `toolbar_groups` feature is designed specifically for the main toolbar.
 
I would recommend one of these as a potential workaround:

 - Custom Pop-Up on Button Click: You can potentially create a custom button for the `quickbars_selection_toolbar` that, when clicked, pops up a custom-styled menu (which you'd implement yourself) that looks and behaves like the `toolbar_groups menu`. This would be quite involved and would require working with TinyMCE's API to both create custom buttons and handle their events.

 - Custom CSS & Tooltips: Although not the same as having grouped buttons in a pop-up, you can visually and functionally mimic a "group" by using CSS and tooltips. For instance, certain buttons can be styled differently or with a specific margin to indicate they're part of a group, and using tooltips can provide a description. It won't have the same interactive feel as a button group but can be an intermediate solution.

- If the grouped buttons are critical for your users, consider emphasizing the use of the main toolbar (where `toolbar_groups` is supported) over the quickbars for certain tasks. This can be an interim solution while you explore more complex custom solutions.

**Reply by GrtFS** [reply]
Thank you for your reply and clear explanation Mrina!

I ended with your 3rd option, but your 1st is interesting and I shall look into re-thinking it with the 1st option applied.

Thanks again. 😊
